ground/src/main/java/com/google/android/ground/repository/LocationOfInterestRepository.kt

```diff
@@ -127,4 +124,33 @@ constructor(
       MutationEntitySyncStatus.IN_PROGRESS,
       MutationEntitySyncStatus.FAILED
     )
+
+  /** Returns a flowable of all [LocationOfInterest] for the currently active [Survey]. */
+  fun getAllLocationsOfInterestOnceAndStream(): Flowable<Set<LocationOfInterest>> =
```

Comment:
Can we update this to only look at the offline LOIs? I'm working on code that will update those in the background, so we no longer need to subscribe to updates from Firestore here; instead we can subscribe to changes in the local Room store.

Reply:
This does return only offline LOIs. Should I rename it to something else to make it more explicit?

Reply:
It's ok for now, I'll update it in my upcoming PR. Thanks!